Modules/Invoices/Tests/Unit/InvoiceServiceTest.php

```
@@ -431,4 +431,152 @@ public function it_returns_zero_days_overdue_for_non_overdue_invoice(): void
+        $client = \Modules\Crm\Models\Client::factory()->create();
+        Invoice::factory()->count(10)->create([
+            'client_id' => $client->client_id,
+        ]);
+
+        /** Act */
+        $result = $this->service->getAllWithRelations(['client'], null, 5);
+
+        /** Assert */
+        $this->assertEquals(5, $result->perPage());
+    }
+}
```

PHPMD (2.15.0)

The class InvoiceServiceTest has 29 non-getter- and setter-methods. Consider refactoring InvoiceServiceTest to keep number of methods under 25. (undefined)

(TooManyMethods)